backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/resource/ResourceService.java

```diff
@@ -0,0 +1,44 @@
+package de.bund.digitalservice.ris.norms.adapter.output.resource;
+
+import de.bund.digitalservice.ris.norms.application.port.output.LoadPortalPublishingAllowListPort;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service for loading data from classpath resources.
+ */
+@Service
+@Log4j2
+public class ResourceService implements LoadPortalPublishingAllowListPort {
+
+  private final List<String> publishingAllowlist = new ArrayList<>();
```

Comment:
If this list should not be modified after it has been created on start-up in the constructor a.k.a be immutable we should use something else, specially not and `ArrayList`. 

And what `Files.readAllLines()` is returning is also a mutable list.

What about wrapping the result of `Files.readAllLines()` into an immutable list like `List.copyOf() `?